### DISCUSSION TITLE
An enhanced version of the App in media

### DISCUSSION THREAD

**Discussion by ioqbit** [spacedriveapp/spacedrive#2109] [General]
https://en.eagle.cool

Eagle can collect scattered pictures, videos, texts, links, models and other files together, and can install plug-ins to expand its functions, such as: viewing the shooting location of the photo on the map, filtering pictures according to color, convenient and fast search and Label.

I think he has something worth learning from
![image](https://github.com/spacedriveapp/spacedrive/assets/66349171/3d4046e0-38c8-42e3-8e02-d22bedc87d9d)
![image](https://github.com/spacedriveapp/spacedrive/assets/66349171/82a2cbd6-a47c-4902-8744-736c382a3fb2)

**Comment by utkubakir** [comment]
Most of what eagle can do is either implemented or better versions are in our roadmap to be honest. For example we have EXIF data for all the images (it's just we don't show a map (yet) and only show latitude/longitude)
